string reference
geometry_msgs/Twist twist
float32 grasp
float32 release
bool waypoint
